openmls/src/treesync/node/leaf_node/capabilities.rs

```diff
@@ -93,6 +93,14 @@ impl Capabilities {
         }
     }
 
+    // XXX: I don't know why rustdoc thinks the builder is private (it's pub), but let's just work
```

Comment:
It is private. This entire module is private, hence the `CapabilitiesBuilder` as well as it's not re-exported as public anywhere.

Reply:
I don't get it. The [Rustdoc documentation](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html) says

> You can refer to anything in scope, and use paths, including Self, self, super, and crate.

and

> Links are resolved in the scope of the module where the item is defined [...].

The reference is in the same module and the thing that is referenced. The referenced item itself is pub. The contents of the module in question are even reexported as pub(crate).

Reply:
I re-exported `CapabilitiesBuilder` in prelude.